### PR TITLE
clarified question mark in property names

### PR DESCRIPTION
**Pull request by sichvoge**
according to #317

### CODE REVIEW

File: versions/raml-10/raml-10.md [hunk 1]
```diff
@@ -529,6 +525,53 @@ types:
...
 
+In order to achieve a more "object oriented" experience, a series of shortcuts are available (see [Shortcuts and Syntactic Sugar](#shortcuts-and-syntactic-sugar)). The example below shows a common idiom:
+
+```yaml
+#%RAML 1.0
+title: My API With Types
+types:
+  Person:
+    properties:
+      name: string # equals to `type: string`
```
Comment (usarid) [inline versions/raml-10/raml-10.md]: Clarify that the *value* is equivalent to `type: string`, or that the *line* `{ name: string }` is equivalent to `{ name: { type: string } }`.

File: versions/raml-10/raml-10.md [hunk 2]
```diff
@@ -504,10 +504,6 @@ Properties of object types are defined using the OPTIONAL **properties** propert
 
 In addition to the properties available in normal type declarations, properties can specify whether they are required and provide an optional default value.
 
-Note:
-
-When an Object Type does not contain the "properties" property, the object is assumed to be unconstrained. That means, it may contain any properties of any type.
-
```
Comment (usarid) [inline versions/raml-10/raml-10.md]: I'm confused by this: why do we want to lose this statement about the "properties" property?

Also, can't every type allow the `default` facet?

Comment (sichvoge) [inline versions/raml-10/raml-10.md]: Agree, looking at current RC1 implementation; `default` is available for every type declaration. @petrochenko-pavel-a can you comment.

Comment (petrochenko-pavel-a) [inline versions/raml-10/raml-10.md]: As for me it is a good to have `default` available for every type definition.

**Pull request merged**